docs/source/conf.py

```diff
@@ -35,6 +35,12 @@
 release = __version__
 version = __version__
 
+rst_prolog = """.. attention::
```

Comment:
I don't think it's worth our time to redo all of the screenshots. Mostly, the web title would need to change in every single one.
The only other aspects are spectral -> array and the removal of a few non-essential fields in ``Instrument`` and ``BioSample``.